projects/kubernetes/kube-master.yml

```diff
@@ -51,18 +51,19 @@ services:
      - /dev:/dev
      - /etc/resolv.conf:/etc/resolv.conf
      - /lib/modules:/lib/modules
+     - /run:/run:rshared,rbind
```

Comment:
The problematic paths in #2131 were `/var/run/secrets/blahblah` which due to the symlink are now actually `/run/secrets/blahblah`. This path needs to exist in both the kubelet and docker containers.

I didn't actually test it but due to that I think having `/run` would break things. I could try it if you want though.